### PR TITLE
Added books for COBOL, FORTRAN and GraphQL free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by adarsh-jha-dev**
## What does this PR do?
Add resources

### Description
I have added the following three books 

- [Learning COBOL](https://riptutorial.com/Download/cobol.pdf)
- [Programming in Fortran 95](https://personalpages.manchester.ac.uk/staff/david.d.apsley/lectures/fortran/fortranA.pdf) -David Apsley
- [GraphQL or Bust](https://nordicapis.com/wp-content/uploads/GraphQL-or-Bust-v2.2.pdf)
### Why is this valuable (or not)?
These are very popular books for the mentioned topics and were not present in the list.

### How do we know it's really free?
I have verified by visiting the provided links for the books.

### For book lists, is it a book? For course lists, is it a course? etc.
These are books.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -595,6 +595,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [COBOL Programming Fundamental](http://yusman.staff.gunadarma.ac.id/Downloads/files/33460/COBOL_Programming_Fundamental.pdf) (PDF)
 * [Enterprise COBOL for z/OS documentation library](http://www-01.ibm.com/support/docview.wss?uid=swg27036733)
 * [ILE COBOL Programmer's Guide](https://www.ibm.com/docs/de/ssw_ibm_i_74/pdf/sc092539.pdf) (PDF)
+* [Learning COBOL](https://riptutorial.com/Download/cobol.pdf) (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:598]: Please read docs/CONTRIBUTING.md for guidelines on riptutorial attribution.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -793,6 +794,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Fortran 90 Tutorial](https://web.stanford.edu/class/me200c/tutorial_90/) - Sarah T. Whitlock, Paul H. Hargrove, Stanford University (HTML)
 * [Fortran information & resources](https://www.fortranplus.co.uk/fortran-information/) - fortranplus.co.uk (HTML)
 * [FORTRAN Performance Tuning co-Guide (1998)](https://www.ibiblio.org/pub/languages/fortran/unct.html) - Timothy C. Prince (HTML)
+* [Introduction to Fortran by David Apsley](https://personalpages.manchester.ac.uk/staff/david.d.apsley/lectures/fortran/fortranA.pdf) -  David Apsley (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:797]:
```suggestion
* [Fortran](https://personalpages.manchester.ac.uk/staff/david.d.apsley/lectures/fortran/fortranA.pdf) -  David Apsley (PDF)
```
Don't invent titles! Use the title given by the resource.

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -842,6 +844,7 @@ Books on general-purpose programming that don't focus on a specific language are
 ### GraphQL
 
 * [Fullstack GraphQL](https://github.com/GraphQLCollege/fullstack-graphql)
+* [GraphQL or Bust](https://nordicapis.com/wp-content/uploads/GraphQL-or-Bust-v2.2.pdf) (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:847]:
```suggestion
* [GraphQL or Bust](https://nordicapis.com/wp-content/uploads/GraphQL-or-Bust-v2.2.pdf) - Nordic APIs (PDF)
```
Please add author names where appropriate.

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by adarsh-jha-dev** [comment]
I have removed that rip tutorial's book and replaced with another book

**Comment by adarsh-jha-dev** [comment]
Hey @eshellman , i have implemented the required  changes, so you can have a look at it.

**Comment by adarsh-jha-dev** [comment]
Hey @eshellman , I request you to please have a look at the changes and let me know if these changes can be merged?

**Comment by eshellman** [comment]
the review process rewards patience.

**Comment by adarsh-jha-dev** [comment]
> the review process rewards patience.

really sorry for being impatient, thanks for merging.

**Pull request merged**